utils/sysinfo/sysinfo.go

```diff
@@ -523,3 +537,31 @@ func GetSocketFromCPU(topology []info.Node, cpu int) int {
 	}
 	return -1
 }
+
+func getNodeStats(sysFs sysfs.SysFs, nodeDir string, statType string) (map[string]int, error) {
+	nodeStatPath := filepath.Join(nodeDir, statType)
+	statsLines, err := sysFs.GetNUMAStats(nodeStatPath)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(statsLines) == 0 {
+		return nil, nil
+	}
+
+	stats := make(map[string]int)
+	for _, line := range statsLines {
+		if nodeStatRegExp.Match([]byte(line)) {
+			words := strings.Split(line, " ")
+			value, err := strconv.Atoi(words[1])
```

Comment:
Please check length of  `words` before accessing an element with index 1.

Reply:
This is unnecessary as nodeStatRegExp already does.